src/views/partials/household.twig

```diff
@@ -0,0 +1,38 @@
+<div class="household-data">
+  <h2>Household</h2>
+  <p>Enter values into the fields below for the Household Level data for the Household Composition.</p>
+
+  <label>zip</label>
```

Comment:
I think zip has to be removed from the request or else the API will return a 4xx

Reply:
you are absolutely right.